bundles/specmate-connectors/bnd.bnd

```diff
@@ -1,5 +1,6 @@
 
 -buildpath: \
+	com.google.guava,\
```

Comment:
guava is one of the few libraries that we use in two versions in different bundles - it would be a good idea to specify a version here (to avoid unpleasant suprise when a differnt one is picked)